### DISCUSSION TITLE
How to establish an indirect pharmacodynamic protein in Mobi and observe it?

### DISCUSSION THREAD

**Discussion by CECI-LALA** [Open-Systems-Pharmacology/Forum#1677] [Questions & Problems]
Hello!
If a protein is an indirect pharmacodynamic site, not directly binding to the drug but influenced by it, should this protein be initially created in the individual within PK-Sim and then transferred to Mobi? Or should it be directly established in Mobi? 

If the protein is present in secretions, how should I define it and observe it? 

Are there any literature or manuals mentioning this?
If anybody has successfully used it for the same purpose, please let me know.

Thanks very much for any reply.

**Comment by StephanSchaller** [comment]
Is it a circulating protein? It depends on if you require the protein to be distributed on a whole-body level or just locally.
If whole-body, criculating, and if it gets secreted, implement it in PKSim and add secretion and any dynamics in MoBi.

**Comment by CECI-LALA** [comment]
thank  you very much @StephanSchaller！
In fact, there are two types of indirect pharmacodynamic proteins that I need to observe: one circulates throughout the body, and the other is locally secreted.
I am currently focusing on the pharmacodynamic proteins that are locally secreted, which are protein complexes excreted in feces.
I have already connected them in "Individual" in PK-Sim and transferred them to Mobi. I added the production and metabolism components in the reaction, and included them in the "concentration in feces" section of the molecule observer. 
However, I cannot find them in "define settings" and can only see my drugs. Am I doing it right?
<img width="608" alt="8781471d5faf0baaa7244c9031518ae" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/127425755/474674bf-4099-48b4-b159-9ee540116987">
<img width="278" alt="f654071f98542f7e2f5622d55b7558a" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/127425755/ae03030f-219d-44f0-aa80-3d6fc0aeee3a">

However, there is still an issue bothering me. 
This secreted protein is commonly measured in feces using the unit mg/kg (ug/g). Even if I can observe it in feces, I am unable to capture its correct dimensional representation. This is because feces in the spatial structure only has volume. 
Could you provide some advice on addressing this concern? I would greatly appreciate it
<img width="501" alt="e42b03583f354d8dbe812e0fa531883" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/127425755/154de8c0-63e7-4f06-973d-26298a03f98d">
